src/components/primitives/FormattedNumber.tsx

```diff
@@ -8,11 +8,12 @@ import { OverridableStringUnion } from '@mui/types';
 interface CompactNumberProps {
   value: string | number;
   visibleDecimals?: number;
+  roundDown?: boolean;
 }
 
 const POSTFIXES = ['', 'K', 'M', 'B', 'T', 'P', 'E', 'Z', 'Y'];
 
-function CompactNumber({ value, visibleDecimals = 2 }: CompactNumberProps) {
+function CompactNumber({ value, visibleDecimals = 2, roundDown }: CompactNumberProps) {
```

Comment:
I'd make this default to `roundDown = false`

Reply:
It doesn't need a default since undefined is truthy and will evaluate to false